packages/backend/src/models/AnalyticsModel.ts

```diff
@@ -0,0 +1,53 @@
+import { Knex } from 'knex';
+import {
+    AnalyticsChartViewsTableName,
+    AnalyticsDashboardViewsTableName,
+} from '../database/entities/analytics';
+
+type Dependencies = {
+    database: Knex;
+};
+export class AnalyticsModel {
+    private database: Knex;
+
+    constructor(dependencies: Dependencies) {
+        this.database = dependencies.database;
+    }
+
+    async countChartViews(chartUuid: string): Promise<number> {
+        const [{ count }] = await this.database(AnalyticsChartViewsTableName)
+            .count('chart_uuid')
+            .where('chart_uuid', chartUuid);
+
+        return Number(count);
+    }
+
+    async addChartView(chartUuid: string, userUuid: string): Promise<void> {
+        await this.database(AnalyticsChartViewsTableName).insert({
+            chart_uuid: chartUuid,
+            user_uuid: userUuid,
+            context: {},
+        });
+    }
+
+    async countDashboardViews(dashboardUuid: string): Promise<number> {
+        const [{ count }] = await this.database(
+            AnalyticsDashboardViewsTableName,
+        )
+            .count('dashboard_uuid')
+            .where('dashboard_uuid', dashboardUuid);
+
+        return Number(count);
```

Comment:
count from db returns a string , but the type is (string | number ), I want a number, but I can't use parseInt because it is not valid with (string | number) type